Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+pipeline {
+	agent any
+	options {
+		buildDiscarder(logRotator(numToKeepStr:'10'))
+	}
+	stages {
+		stage('Prepare') {
+			steps {
+				git url: 'https://github.com/eclipse/wildwebdeveloper.git'
+				cleanWs()
+				checkout scm
+			}
+		}
+		stage('Build') {
+			steps {
+				wrap([$class: 'Xvnc', useXauthority: true]) {
+					withEnv([]) { // Which environment variable we should use for node ?
+						withMaven(maven: 'apache-maven-latest', jdk: 'jdk1.8.0-latest', mavenLocalRepo: '.repository') {
+							sh 'mvn clean verify -Dmaven.test.error.ignore=true -Dmaven.test.failure.ignore=true -PpackAndSign'
+						}
+					}
+				}
+			}
+			post {
+				success {
+					junit '*/target/surefire-reports/TEST-*.xml' 
+				}
+			}
+		}
+		stage('Deploy') {
+			when {
+				branch 'master'
+				// TODO deploy all branch from Eclipse.org Git repo
+			}
+			steps {
+				// TODO compute the target URL (snapshots) according to branch name (0.5-snapshots...)
+				sh 'rm -rf /home/data/httpd/download.eclipse.org/wildwebdeveloper/snapshots'
```

Comment:
I'm not sure about those path on the new CI instance. But let's just keep them at the moment and fix them later if it happens they're wrong.